packages/flutter/test/material/switch_test.dart

```diff
@@ -21,6 +21,8 @@ import '../rendering/mock_canvas.dart';
 import '../widgets/semantics_tester.dart';
 
 void main() {
+  final ThemeData theme = ThemeData(brightness: Brightness.light);
```

Comment:
I believe this can just be:

```suggestion
  final ThemeData theme = ThemeData();
```